tools/tdbg/tdbg_commands.go

```diff
@@ -562,6 +562,54 @@ func newAdminDLQCommands(
 				return ac.MergeMessages(c)
 			},
 		},
+		{
+			Name:        "job",
+			Usage:       "Run admin operation on DLQ Job",
+			Subcommands: newAdminDLQJobCommands(dlqServiceProvider),
+		},
+	}
+}
+
+func newAdminDLQJobCommands(
+	dlqServiceProvider *DLQServiceProvider,
+) []*cli.Command {
+	return []*cli.Command{
+		{
+			Name:        "describe",
+			Aliases:     []string{"d"},
+			Usage:       "Get details of the DLQ job with provided job token",
+			Description: "This command will get details of the DLQ job with provided job token if using v2",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:  FlagJobToken,
+					Usage: "ID of the DLQ job",
```

Comment:
nit: I would say token instead of ID, and specify that this is in the response of the merge and purge commands.